tests/integration/security/ca_custom_root/multi_root_test.go

```diff
@@ -37,46 +38,41 @@ func TestMultiRootSetup(t *testing.T) {
 
 			for _, cluster := range t.Clusters() {
 				t.NewSubTest(fmt.Sprintf("From %s", cluster.StableName())).Run(func(t framework.TestContext) {
-					verify := func(ctx framework.TestContext, src echo.Instance, dest echo.Instance, s scheme.Instance, success bool) {
+					verify := func(ctx framework.TestContext, from echo.Instance, to echo.Instances, s scheme.Instance, success bool) {
 						want := "success"
 						if !success {
 							want = "fail"
 						}
-						name := fmt.Sprintf("server:%s[%s]", dest.Config().Service, want)
+						name := fmt.Sprintf("server:%s[%s]", to[0].Config().Service, want)
 						ctx.NewSubTest(name).Run(func(t framework.TestContext) {
 							t.Helper()
-							opt := echo.CallOptions{
-								Target:   dest,
+							opts := echo.CallOptions{
+								Target:   to[0],
 								PortName: HTTPS,
-								Address:  dest.Config().Service,
+								Address:  to[0].Config().Service,
 								Scheme:   s,
 							}
-							checker := connection.Checker{
-								From:          src,
-								Options:       opt,
-								ExpectSuccess: success,
-								DestClusters:  t.Clusters(),
-							}
-							checker.CheckOrFail(t)
+							opts.Check = check.And(check.OK(), scheck.ReachedClusters(to, &opts))
```

Comment:
super nit: define inline in the struct creations

Reply:
Can't ... the `ReachedClusters` requires the struct.